teuthology/misc.py

```diff
@@ -512,15 +512,18 @@ def gen_addresses():
     return fsid
 
 
-def write_file(remote, path, data):
+def write_file(remote, path, data, bs=None, offset=None, sync=False):
     """
     Write data to a remote file
 
     :param remote: Remote site.
     :param path: Path on the remote being written to.
     :param data: Data to be written.
+    :param bs: The block size to be used.
+    :param offset: The number of bytes to seek to in file.
+    :param sync: Sync file.
     """
-    remote.write_file(path, data)
+    remote.write_file(path, data, bs=bs, offset=offset, sync=sync)
```

Comment:
This method is left for backwards compatibility, for newer code, please, use only remote context function, so I mean, please, don't change this method.